src/main/java/com/roadtriprunner/RoadTripRunner/controllers/AuthenticationController.java

```diff
@@ -86,37 +86,37 @@ public String processRegistrationForm(@ModelAttribute @Valid RegisterFormDTO reg
         return "redirect:/planATrip";
     }
 
-    @GetMapping("/login")
+    @GetMapping("/")
     public String renderLoginForm(Model model) {
         model.addAttribute(new LoginFormDTO());
         model.addAttribute("title", "Login");
-        return "login";
+        return "index";
```

Comment:
because this was changed to the index page.... the Login button in the top right corner needs to link to homepage instead of /login